boot/zephyr/Kconfig.serial_recovery

```diff
@@ -56,6 +56,12 @@ config BOOT_MAX_LINE_INPUT_LEN
 	help
 	  Maximum length of commands transported over the serial port.
 
+config BOOT_SERIAL_USE_PIN_DETECT
```

Comment:
worth to add an option for "exclusively detect whether the application requested the DFU mode"